overlord/hookstate/ctlcmd/helpers.go

```diff
@@ -238,6 +244,114 @@ func runServiceCommand(context *hookstate.Context, inst *servicestate.Instructio
 	}
 }
 
+func validateSnapAndCompsNames(names []string, ctxSnap string) ([]string, error) {
+	var allComps []string
+	for _, name := range names {
+		snap, comps := snap.SplitSnapInstanceAndComponents(name)
+		// if snap is present it must be the context snap for the moment
+		if snap != "" && snap != ctxSnap {
+			return nil, fmt.Errorf("cannot install snaps using snapctl")
```

Comment:
This `fmt.Errorf` could be an `errors.New`

Reply:
Good catch, changed